rand_core/src/lib.rs

```diff
@@ -361,7 +361,7 @@ impl<'a, R: RngCore + ?Sized> RngCore for &'a mut R {
     }
 }
 
-#[cfg(any(feature="std", feature="alloc"))]
+#[cfg(feature="alloc")]
```

Comment:
Maybe I shouldn't have merged so quickly; I just realised this is in a separate crate which does not have the same rule. This is technically wrong; any crate depending on rand_core with 'std' but not 'alloc' and trying to use this impl rule will now fail if there isn't also a dependency on rand. I'm testing adding a similar dependency in rand_core; seems to work.

Reply:
Oops, you aye right. I messed up my commit and missed the change to the Corgo.toml of rand_core